requirements.txt

```diff
@@ -12,4 +12,4 @@ ninja
 peft>=0.8.0
 datasets>=2.15.0
 fire
-
+wheel
```

Comment:
wheel is for packaging. should this be in a development dependencies ?

Reply:
+1 ^

Reply:
OK, I have it in a separate [dev] package, and have removed the flash_attn txt file.  I think this is simpler and works consistently.  🤞